packages/ui/src/helpers/accountSettings/validator.ts

```diff
@@ -159,12 +159,16 @@ export const runFieldValidators = ({
 }): string[] => {
   if (!value) return [];
 
-  return validators.reduce((prevErrors, validatorSpec) => {
-    const { validator, validationMode, message } = validatorSpec;
-
-    if (shouldValidate({ validationMode, eventType, hasBlurred })) {
-      const hasError = !validator(value);
-      return hasError ? [...prevErrors, message] : prevErrors;
-    }
-  }, []);
+  return validators.reduce(
```

Comment:
Correctly if I'm wrong, but looks like just types changed here?

Reply:
Added types and a return statement to the end of the `validators.reduce` function to ensure that it always returns a value